scripts/idl/generators/bridge/__init__.py

```diff
@@ -44,22 +44,15 @@ def get_field_info(definition: Field, cluster: Cluster, idl: Idl):
     context = create_lookup_context(idl, cluster)
     actual = ParseDataType(definition.data_type, context)
 
-    orig = actual
-    is_enum = type(actual) == IdlEnumType
-
-    if type(actual) == IdlEnumType:
-        actual = actual.base_type
-    elif type(actual) == IdlBitmapType:
+    if type(actual) == IdlEnumType or type(actual) == IdlBitmapType:
```

Comment:
Please provide a unit test that makes sure this does not regress (and also shows what kind of output to expect).

Unit tests are made via `golden images` of expected generated code. See `scripts/idl/tests/`

Reply:
I pushed a golden image with a bitmap test. This shows codegen pushing bitmap8 datatype.